gammapy/data/event_list.py

```diff
@@ -653,6 +653,47 @@ def plot_time_map(self, ax=None):
 
         return ax
 
+    def plot_theta2_distribution(self, ax=None, number_bins=50):
+        """Plot the theta2 distribution of the events.
+        A pointing direction can be given in the same units as the event list as (ra,dec).
+        If None is given, the average of incoming directions of the events is taken.
+
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes` or None
+            Axes
+        number_bins: integer
+        pointing_radec: Tuple of floats or None
+
+        Returns
+        -------
+        ax : `~matplotlib.axes.Axes`
+            Axes
+
+        """
+        import matplotlib.pyplot as plt
+
+        ax = plt.gca() if ax is None else ax
+
+        events_ra = self.table[:]['RA']
+        events_dec = self.table[:]['DEC']
+
+        pointing_ra = self.pointing_radec.ra.value
+        pointing_dec = self.pointing_radec.dec.value
+
+        theta2 = (events_ra - pointing_ra)**2 + (events_dec - pointing_dec)**2
```

Comment:
There's a few issues with how you compute `theta2`: I think `table[:]` makes a copy, so is very inefficient. Since you don't need a copy, just use `self.table['RA']` etc. More importantly, I think we should use separation on the sphere, the formula you have doesn't give accurate results, especially if the run is pointing near the poles (i.e. dec = +90 or =90 deg).

So instead I would suggest that you use http://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html#astropy.coordinates.SkyCoord.separation. Actually, this is already available via http://docs.gammapy.org/en/latest/api/gammapy.data.EventList.html#gammapy.data.EventList.offset, i.e. you can just access that property.

Note that your docstring currently lists `pointing_radec` as an argument, but you don't really take that as input. So either remove that from the docstring, or keep a `dist_center=None` argument to let the caller choose the center to use, and default to `self.pointing_radec`.